traffic/algorithms/navigation.py

```diff
@@ -716,11 +722,12 @@ def go_around(
         if airport is None:
             return None
 
+        attempts = self.aligned_on_ils(airport, **kwargs)
         # you need to be aligned at least twice on a rway to have a GA:
-        if len(self.aligned_on_ils(airport, **kwargs)) < 2:
+        if len(attempts) < 2:
             return
 
-        first_attempt = next(self.aligned_on_ils(airport, **kwargs), None)
+        first_attempt = next(attempts, None)
```

Comment:
here `first_attempt` is always `None` since it has been consumed above

Reply:
I don't think it has yet been consumed...